pages/accountLists/[accountListId]/contacts.page.tsx

```diff
@@ -30,39 +34,15 @@ const ContactsPage: React.FC = () => {
       <Head>
         <title>MPDX | {t('Contacts')}</title>
       </Head>
-      <div style={{ display: 'flex' }}>
-        <div style={{ margin: 5 }}>
-          <h2>Filters</h2>
-          <button onClick={() => loadContactFilters()}>Load Filters</button>
-          {contactFiltersError && (
-            <p>Error: {contactFiltersError.toString()}</p>
-          )}
-          {contactFiltersLoading ? (
-            <p>Loading Filters</p>
-          ) : !contactFilters?.contactFilters ? (
-            <p>No filters</p>
-          ) : (
-            <ul>
-              {contactFilters.contactFilters.map(({ id, name }) => (
-                <li key={id}>{name}</li>
-              ))}
-            </ul>
-          )}
-        </div>
-        <div style={{ margin: 5 }}>
-          <h2>Contacts</h2>
-          {error && <p>Error: {error.toString()}</p>}
-          {loading ? (
-            <p>Loading</p>
-          ) : !data?.contacts?.nodes ? (
-            <p>No data</p>
-          ) : (
-            data.contacts.nodes.map((contact) => (
-              <ContactRow key={contact.id} contact={contact} />
-            ))
-          )}
-        </div>
-      </div>
+      <Contacts
```

Comment:
Some thoughts about how to structure components here:
1. I think the `ContactsPage` component should be the one to split the page into 2 columns.
2. We can make `ContactsPage` store the selected filters state (future work) and pass those between the left filters pane and the right contacts list.
3. We should make the left filters component and the right contact list contain the GraphQL queries. I don't think there's any reason to have them at the top level and pass everything down.